design/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!--Google font-->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!--Style sheet-->
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Mobile portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h1>My Logo</h1>
+        <img src="./assets/Union.svg" alt="burger-menu" />
+      </nav>
+    </header>
+    <main>
+      <section id="project-props">
+        <!-- <div class="section-main-img"></div> -->
+        <img
+          class="project-img"
+          src="./assets/Snapshoot Portfolio.png"
+          alt=""
+        />
+        <h2 class="project-name">Multi-Post Stories</h2>
+        <div class="project-details">
+          <h5>CANOPY</h5>
+          <img src="./assets/Counter.svg" alt="dot" />
+          <h5>Back End Dev</h5>
+          <img src="./assets/Counter.svg" alt="dot" />
+          <h5>2015</h5>
+        </div>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="project-used-techs">
+          <h6>html</h6>
+          <h6>css</h6>
+          <h6>javascript</h6>
+        </div>
+        <button type="button" class="project-button">See Project</button>
+      </section>
+    </main>
```

Comment:
- [ ] The content of this section is quite different from what the project required. Kindly look into the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) and change the contents to the required content. I wish you the best. :pray: